scripts/CodeCoverage.py

```diff
@@ -133,7 +133,7 @@ def get_gcovr():
 
 def get_gcov():
     if PROBE_BUILD:
-        return '--gcov-executable /usr/local/probe/bin/gcov'
+        return '--gcov-executable /usr/bin/gcov'
```

Comment:
This would point to gcov for version `4.8.5` while the compiler in `/usr/local/gcc/bin` points to a newer version of gcc

Reply:
I.e. shouldn it stay `/usr/local/gcc/bin/gcov`?